spelling_wordlist.txt

```diff
@@ -0,0 +1,87 @@
+ABI
+Balancer
+Ethereum
+Hoare
+OOM
+Prover
+Secureum
+Todo
+axiomatization
+axiomatized
+bitwise
+blockchain
+boolean
+bootcamp
+bytecode
+conjuncted
+dereference
+dereferences
+enum
+enums
+envfree
+executable
+executables
+getter
+getter
```

Comment:
Should this second "getter" instead be "getters" given the pattern elsewhere in this list?

Reply:
fixed